pom.xml

```diff
@@ -102,7 +102,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-cps</artifactId>
-            <version>2.41</version>
+            <version>2.48-20180409.122132-1</version>
```

Comment:
TODO link to upstream

Reply:
Linked in the PR description, no plans to merge w/o relese